submissions/ksalpern/Hooli-style-Popup/index.html

```diff
@@ -226,7 +225,7 @@
                           <a href="#" class="popup__more-icon">
                             <img
                               src="./popup-icons/tanslation.png"
-                              alt="Translation"
+                              alt=""
                               class="icon"
                             />
                             <span class="popup__more-name">Translate</span>
```

Comment:
The text must be inside his container
![image](https://user-images.githubusercontent.com/16196199/182795674-ade7092b-170f-4c8f-b85a-ca65ad907d2c.png)

Reply:
![image](https://user-images.githubusercontent.com/101118611/182827879-7e665723-4763-4853-bf67-59c1cdb57903.png)

Please take a look how it is now)

Reply:
I checked and found that the problem was on my side: I'm using Ubuntu + Firefox, and the browser uses the default font. So on my system + browser, I have this issue. In real work, you must create UI that looks the same on all systems and browsers. Fix this or not is up to you)

Reply:
Got it. So I need to put default font and it will be shown on each browser the same, right?

Reply:
Yeah, you need to set the font for your application.